File: java/Day4/codes/demo10/src/com/sunbeam/Person.java
package com.sunbeam;
//person class 
public class Person {
	private String name; 
	private int age; 
	public Person() {
		System.out.println("public Person()");
	}
	public Person(String name, int age) {
		System.out.println("public Person(String name, int age) ");
		this.name = name;
		this.age = age;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public int getAge() {
		return age;
	}
	public void setAge(int age) {
		this.age = age;
	}
	public void display( ) {
		System.out.println("Name : "+this.name);
		System.out.println("Age : "+this.age);
		
	}
	
}

File: Pune/Ms.Net/Dotnet/day02/day02/Day02Soln/07_OopInheritance/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _07_OopInheritance
{
    internal class Person
    {
        public string Name {  get; set; }
        public int Age { get; set; }

        public Person() : this("", 0)
        {
        }
        public Person(string name, int age)
        {
            Name = name;
            Age = age;
        }
        public virtual void Display()
        {
            Console.WriteLine($"Person: Name={this.Name}, Age={this.Age}");
        }
    }
}
